Dockerfile

```diff
@@ -60,7 +63,7 @@ gpgkey=https://dl-ssl.google.com/linux/linux_signing_key.pub' \
   && driver=$(curl -s "https://googlechromelabs.github.io/chrome-for-testing/last-known-good-versions-with-downloads.json" | jq --arg majorVersion "$CHROME_VERSION" -r '.channels.Stable | select(.version | startswith($majorVersion | tostring)).downloads.chromedriver[] | select(.platform == "linux64") | .url') \
   && wget -N --progress=dot:giga "$driver" -O /tmp/chromedriver-linux64.zip \
   && unzip -j /tmp/chromedriver-linux64.zip -d /tmp \
-  && rm /tmp/chromedriver-linux64.zip \
+  && rm -f /tmp/chromedriver-linux64.zip \
```

Comment:
why was this needed :thinking: , the file should always exist, no?

Reply:
I guess it's just cleanup to not leave the tarball in the image. We need the binaries extracted in the command above, not the dist tarball.

Reply:
I was asking about the `-f`. The `rm` was there since before. `-f` will cause command not to fail if deleting failed or file didn't exist.

Reply:
yeah, honestly, I don't know why I added it, it could have easily been by mistake and not because it was needed.